exercises/changing-text.md

```diff
@@ -0,0 +1,30 @@
+# Changing text
+
+This exercise introduces **Html blocks** to find and modify the text of html elements.
+
+Let's start with the html below in the html area. It represents a sentence with gaps (also known as [Mad Libs](https://en.wikipedia.org/wiki/Mad_Libs)). An example of a mad lib sentence would be the following:
+
+>  The dog saw the ___ cat
```

Comment:
I've updated this exercise to introduce an actual gap into the sentence so students can find a way of actual removing the gap via a Block. However, I can change this back to the original problem if I've drifted too far from the problem's original intent.